db/viewDefinitions.js

```diff
@@ -11,6 +11,7 @@ function getViewDefinitions(db) {
     video: [
       `is_excluded='0'`,
       `is_censored='0'`,
+      `is_public='1'`,
```

Comment:
I think Atlas filters non public videos by itself in public feeds, but allows to see non public videos in Studio or directly through video URL